documentation/cloud-manager-guide/admin/user-management/edit-user.md

```diff
@@ -0,0 +1,122 @@
+
+To access the *Edit User* tab by directly selecting the user, select the user name from the list of users on the *Admin* *User* tab.
+
+![image description](images/edit-select-username.png)
+
+This brings you to the *Environment Access* tab in a detailed overview of the user. This view lists all the environments and associated roles assigned to the user. From the *Environment Access* tab, select the **Edit User** button to open the *Edit User* tab.
+
+![image description](images/edit-select-edituser.png)
+
+Alternatively, from the *User* tab you can select the **Options** (**...**) menu associated with the user. Select **Edit** from the dropdown menu to bypass the user detailed view and navigate directly to the *Edit User* tab.
+
+![image description](images/edit-users-options.png)
+
+The *Edit User* tab contains all of the same categories and fields as the *Create User* tab. From here you can update the user's details, status, role, and environment assignments accordingly.
+
+![image description](images/edit-user-tab.png)
+
+## Update user details
+
+From the *Edit User* tab, you can update a user's first name, last name, and email address in the *User Details* section. Enter the updated information in the associated field provided and select **Save** to update the user's details.
+
+![image description](images/edit-user-details.png)
+
+## Update user status
+
+There are two ways to update the status of a user, either from the *Admin* *Users* tab or from the *Edit User* tab.
+
+To update a user's status from the *Users* tab, select the **Options** (**...**) menu located next to the user who you would like to deactivate or activate.
+
+![image description](images/edit-option-menu.png)
+
+From the **Options** (**...**) dropdown, select **Deactivate**.
+
+![image description](images/edit-deactivate.png)
+
+You will receive a message requesting confirmation that you would like to deactivate the user. Select **Confirm** to continue and deactivate the user.
+
+![image description](images/edit-deactivate-confirmation.png)
+
+The user's status will change from "Active" to "Inactive"
+
+![image description](images/edit-inactive-status.png)
+
+To update a user's status from the *Edit User* tab, adjust the *Status* toggle in the *User Details* section to the "Active" or "Inactive". Select *Save* to update the user's status.
+
+![image description](images/edit-user-inactive.png)
+
+After selecting save, you will return to the *Users* tab where the user status will update accordingly to the status you selected.
+
+![image description](images/edit-inactive-status.png)
+
+## Update role details
+
+To update the role or environment assignments of a user, select the **Options** (**...**) menu next to the role and environment pairing to be edited. Select **Edit** from the dropdown to enable editing for the role and environment fields.
```

Comment:
Can you access this two ways as well? One from the Options ... menu and another from selecting the user and then selecting Edit User? If yes, we should include that here but maybe just in words and not screenshot the entire process again.